package.json

```diff
@@ -33,7 +33,9 @@
 		"@gisatcz/ptr-atoms": "^1.6.0",
 		"@gisatcz/ptr-core": "^1.7.0",
 		"@gisatcz/ptr-locales": "^1.5.0",
+		"@gisatcz/ptr-state": "^2.1.0",
```

Comment:
You should not add @gisatcz/ptr-state and @gisatcz/visat-components  to this package.